osu.Game.Tests/Database/RealmLiveTests.cs

```diff
@@ -66,19 +66,29 @@ public void TestValueAccessWithOpenContext()
 
                 Task.Factory.StartNew(() =>
                 {
-                    Assert.DoesNotThrow(() =>
+                    // TODO: The commented code is the behaviour we hope to obtain, but is temporarily disabled.
+                    // See https://github.com/ppy/osu/pull/15851
```

Comment:
I'm not super sure about these todos. Given the realm team's answer, is the "desirable" behaviour ever actually going to be within reach? If not, then at that point the code could as well be removed as it serves little purpose.

Alternatively these paths could be shipped off to `[Ignore]`d tests. But I'm still not sure they would give much value.

Reply:
I'll remove it all completely for now.